vidarr-server/src/test/java/ca/on/oicr/gsi/vidarr/server/PriorityByWorkflowTest.java

```diff
@@ -130,15 +130,17 @@ public void testIfResourceIsReleased_thenWorkflowRunIsAddedBackToWaiting() {
     JsonNode higherPriority = mapper.valueToTree(4);
     JsonNode lowerPriority = mapper.valueToTree(2);
 
-    sut.set(workflow, "qwerty", Optional.of(higherPriority));
+    Optional<String> requestErrorHigher = sut.request(workflow, version, "qwert",
```

Comment:
It looks like you've got vidarr ID `qwert` here, and `qwerty` below - was this intentional?

Reply:
Nope, that would not be testing what I wanted. Thank you for catching!